website/data/docs-nav-data.json

```diff
@@ -1,4 +1,34 @@
 [
+  {
+    "title": "What is Packer?",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "intro"
+      },
+      {
+        "title": "Why Use Packer?",
+        "path": "intro/why"
+      },
+      {
+        "title": "Use Cases",
+        "path": "intro/use-cases"
+      }
+    ]
+  },
+  {
+    "title": "HCP Packer",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "hcp-packer"
+      },
+      {
+        "title": "Configuring HCP Packer",
+        "path": "hcp-packer/configure"
```

Comment:
hmm I wonder if this could just be one page. Thoughts? I think there may not be enough content for two subpages (and I hate making users click more than they absolutely have to!)

Also sidebar - isn't there a static link to the HCP packer docs at the bottom of the nav somewhere? I just wanna make sure we get rid of that too since we're making this nice new pointer page.

Finally - YAY FOR ADDING IN THE INTRO CONTENT!!!! I've been meaning to do that so I'm thrilled to see it here.